language/diem-framework/modules/0L/TowerState.move

```diff
@@ -21,6 +21,8 @@ module TowerState {
     use 0x1::VDF;
     use 0x1::Vector;
 
+    const VDF_SECURITY_PARAM: u64 = 2048;
```

Comment:
Why is this 2048? I thought the point was to move 256?

Reply:
This wasn't quite ready for review. First getting the old tests to pass with new APIs, then change the params.